frontend/src/scenes/sceneLogic.ts

```diff
@@ -175,7 +175,9 @@ export const sceneLogic = kea<sceneLogicType>({
         loadScene: (scene: Scene, params: Params) => ({ scene, params }),
         setScene: (scene: Scene, params: Params) => ({ scene, params }),
         setLoadedScene: (scene: Scene, loadedScene: LoadedScene) => ({ scene, loadedScene }),
-        showUpgradeModal: (featureName: string) => ({ featureName }),
+        showUpgradeModal: (featureName: string, featureBenefit: string) => ({
+            featureNameBenefit: [featureName, featureBenefit] as [string, string],
```

Comment:
`featureNameBenefit` is confusing, you're sending both the name and benefit in here, I would change this to something more clear.